src/main/java/com/example/backend/Model/Room.java

```diff
@@ -98,6 +118,15 @@ public void toogleAvail() {
 			this.isBooked = false;
 		else this.isBooked = true;
 	}
-	
-	
+		
+	public boolean sameRoom(Room room) {
+		if (getNumGuest() == room.getNumGuest()
+				&& getLocation().toLowerCase().contains(room.getLocation().toLowerCase())
+				&& (isBooked == room.isBooked())) {
+			return true;
+		}
+		else {
+			return false;
+		}
```

Comment:
Please change method name to be more detailed or put this logic into RoomController.

Reply:
Changed name to equals, will change to something else if need be.